instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -20,15 +20,16 @@
 def _extract_conn_attributes(conn_kwargs):
     """ Transform redis conn info into dict """
     attributes = {
-        "db.type": "redis",
-        "db.instance": conn_kwargs.get("db", 0),
+        "db.system": "redis",
+        "db.name": conn_kwargs.get("db", 0),
     }
     try:
```

Comment:
it looks like this is old code, but an if/else statement checking existence of keys is faster than a try/catch (more work like capture the stack when an exception is thrown)

Reply:
https://docs.python.org/3/glossary.html?highlight=eafp#term-eafp, this is the reason I let this be `try`.. `catch`. I will change it if needed.